src/Behat/Behat/Output/ServiceContainer/Formatter/JUnitFormatterFactory.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Output\ServiceContainer\Formatter;
+
+use Behat\Testwork\Output\ServiceContainer\OutputExtension;
+use Behat\Testwork\Exception\ServiceContainer\ExceptionExtension;
+use Behat\Testwork\Output\ServiceContainer\Formatter\FormatterFactory;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Behat junit formatter factory.
+ *
+ * @author Wouter J <wouter@wouterj.nl>
+ */
+class JUnitFormatterFactory implements FormatterFactory
```

Comment:
I´m aware of compositing over inheritance, but I have difficulties to accept the argument that it introduces a BC break adding it, as the only one to make this final. It`s just a factory not a leave of a complex inheritance tree which would make ugly things if you inherit it. IMHO